### ISSUE TITLE
Upgrade testrpc-sc when ganache floating timestamps issue is fixed

### ISSUE THREAD

**Issue by barakman** [sc-forks/solidity-coverage#252]
I think that you might want to add in the [FAQ document](https://github.com/sc-forks/solidity-coverage/blob/master/docs/faq.md) a note about testing Solidity code which uses `now` (or `block.timestamp`).

From what I've noticed, `solidity-coverage` and `truffle test` may show different results when testing such code. I would assume that this has something to do with "stubs" added by `solidity-coverage`.
If my observation is correct, then it might be worth noting it in the docs.

Thanks

**Comment by cgewecke** [comment]
This is a [bug](https://github.com/trufflesuite/ganache-core/issues/111) in ganache-core with a [pending fix](https://github.com/trufflesuite/ganache-core/pull/443). There shouldn't be any variation in blocktimes between coverage and regular tests - ganache is supposed to set the timestamp once per block.

**Comment by barakman** [comment]
@cgewecke:

In any case, I have observed different behavior between "truffle + ganache-cli" and "solidity-coverage + testrpc-sc", executed with the exact same test-suite.

After some investigation, I concluded that it was tightly-coupled with time-dependent onchain code (i.e., usage of `now`).

If this is indeed a bug in ganache core as you say, will the pending fix apply for testrpc-sc as well?

Thanks
